icons/svgr.svg

```diff
@@ -0,0 +1,11 @@
1uEbAygAA+z75503gQbpIdqBdayXksw7nAAsOjwFbpVHfx3SymYlOxEKlST43v5LMicMgk8mQzqRJpdP4fnhiHscxd959P3fe9xD7D07iEKYXKhxfWOabt9zH5LFpQt/ntL3bueDc09m4YYwkMScKdicKNyRJTPRvhTdbLFerzM/NMzu3wEK5hmqWGWtM0G+reL6HQrDiMDbxlFLh3rPP+kY69yfnTt33nZ+q1iPWCOgNrMUvDFXrTYgTEAGsIA608ugzy8Tl71NujFBuj7Fcz+L5y4S+h7+SVBiQzabI53P0lUpo3+er372bdDbPa17xEnbt2ILvecweP0QpH5BYj4cen+C2ex5hsbLMa158JQ6h1WyeaPN6o0693jiR5eU6y/UWnThBkohca5bBxiQF2ujAI9AgOCIB7Yf16tEnywADWXlMX/ji99tOQ60RUNp2Gk9HazqcPJhAJ4bQ6z3AeaJI+T5Fk2AbR1muzxF5OVrpEq1sPzpMgfJwShMGHplUQKmQ4+Kz9rB9+zjFYhELKEnYOtZHp9MmtooLz97D7vGNzMwtsG/f/hOFlivLLNcbtKOEOE7AGDxxBBrC5hLpxjR9tkrecwS+v3pOoYuJ8bOlxeKmHbN0KUId1uJt2b2Lp6PSX7zjwUyRerODlwtw3RcXHo6UEkzg49CoTkK9tUCrtkCiQowfIishVcDm+mkEKZaqDebLyxw6PEmxkDtxwgx9jRaLcUIUJzSbbeqNJrV6k+VGm1Y7xpmYQDm0TfA7daS9jGc7eEmbMGmQCxzZdEDa1/gKtADdT4etcXj5vvncupMqPAvSKC/wdIhSxY+9/0W3Np+445SxoSKCw7iV4EgsRBZaxtGMHa3EnViOEktsLM6uBHBeiAsyuMIw/uB6VBBirMM5ixKHEn1iPTEWsxIRwfMUknQwlXl0cwmdtCHuICZCYfEVpDxFNvTIh4qcL4RaUAgCiHJ4SijPL7Pppe/7R+AtPAvePR/9ZZ6B6siec+988N47TikVIFB0P4AUALRzBA4QQSuHjyMRRaLBOHA4jLUk7QpJs4IpT0OuhA5SWC9EgjROe5DEqJV4JsZ3q8e1qZfRrWUUFk8pPK3xfQ9fy0o4UXDq3wJoK92Px8ABOCGKImymwOZLXvYZngNvZSeeibBv5Mbvf+Ozr1uqHC/0F7I4HN0vHIJy4AMioEUwitWzrwOLYJTCaE1iHXHcJJ6vYRyIKNAaRJ0QqbFgLQ4LQKA1nufjacETwVfgKfCVrM5F8LvFg2B7HwehPGjU22R3n/3w2OkXfY/nQP5f8dYA9ToTRO+mdp9t27Zt27Zt27Zt27Ztm2lf+jfJf5Lt2fM+vs/frVbF3J1JOmduBLcLwWH3uN6rb6ye2yh5QoeaV1OWmeSXnp9Z+Y/Oi2qbqVypm8tQHmoYsbVMFkd3UYldDqwoDI1aKqBGa5RXtQ9AXc/mwOqqCjlaEd94F7J3mTkAwPh/esDtnUsRHHLVaDL70Yk9FX/+eBcjhs1Md0q1QJGpaUB8uyjDl3NDFTTScIGPEA3dJVHV+HGsfOW3PkqNYiVCUBJUMkBoX6b5vFcEBFGGWQdIoOspgUpG/hvGZFmepS5efTVCAPLjw2v8C2fXLpxyds7Insni2lXWXW4BvwQJTpNOjU0onhGIFpqJRJVX1mdtBr9FYvZQjGVjlAxZ9RaXF3jHe5HIysGipzVJwinEe/FLEJCj69ThAEaEiIDL62bhX0hRpEqG5W2qHJJe3U1khghtqlxPUtZos/nZmkntyOeXTs5oZpazFwTeZmv+cfESDQuZiSCYllqVHGrkNySGYHO/NyS2aoiSmCnkeN0/YMta9Fb5KdtKAPgSIgL47yFahxNLp48+NX3UoNhxnCgxZkVLAMs+3ry48M7CMW1ixLZDJkyiEizYNAjrEb9jbJSwJcoYUdtKAH36KSBJ7U7rvj27k/brmf25kiSwQa/j4P71AylqdVwGoCVCCO252UMQEhRo2X/BjZ3rWjuTpHyao1zV5QDwJlX6pdd3rq/Pf31qszgcSlwzEgitdjHtLyOGubtvgIUEgZ+jgk8qQD+TsDj3/HGDhw65q9VZ7YzT6cWa7k12vLp1JlUCGweiN8DgjPMEIQfIqdnDEFLI9nj9IUmGv+NL1FsGHRnSZnRimxZagxGiRN3WzfOQRAkmi0U9PKvmBXB1wsbpfwwa44wkSVI+j11Ahf88ePeJR/rmXTc3GDOjAQDvw8vnUzw4uqvD28PrW7nePI1ZdvzKRgDWhpiA2ztXIqRInLekyfNH4ADw8MGZJIVx7dBeKx+umlYnSWyren4XXC7YM+S5LWsM4pcrx7NZrBYQTsOM9usJgUuCFeO/8h5ZYzTBoZeJx+3Gt98CklRpfLTa4EnNAfhReH6+dynbvb0bpsVJn2MBgA0hJuDk4skIL7LXbZdwQbv6e3+e3ZMtllULWW/x1l5+uHTMlOnvbOpcf/frI3vy2Z0mcEqe8I9LBNS214vvP38jVrGqF8mvz7E/Xzyb0mMxI3OLnkuKt+7eJyjVp/jtQ4KPTx8CwLsQE3Bj/zZEBAxxk5SfX7/MHsuXb1yZwUNnAOgOACnL1k58dvG0+U92rqpkhAiDzQ45wGHPFwySCI+S+goSklaofaryiBkddvRqufDBqUMFK41f2B/ABEQwyPHVixFR+PDi+arvD2+lazprRbm/dknJ6003d2/pcmPdgt4/HtyOY7GZ1IxRojGvpq4enocIAkfaLB+Tl66+pFCzjlMBfL5zaPdAjyT/AjALkQByZOUiRBRyV6qVUJJEPYDnCARf3r7Ken7l3DH3d6ytrPO6YbTaIPz+BYlwSJCz4P2kBUqtTV2y0noAj+BDojTpObfgYZqaiAb58vkLohIanV5348DOtqdnjx76++G9uAmzZ3+aq2nnaamLlV+ncIQoBvn45jWiA+8e3it+afWCygWbqzn7TUQT/gcNRW9h9S03rwAAAABJRU5ErkJggg=="/>
```

Comment:
In addition, we always try to avoid base64 strings in our SVG images. The reason for this is that base64 strings are images that are not scalable. However, the scalability of all SVG icons is very important as VS Code users do have different screen resolutions and will render the icons in different sizes.